### PR TITLE
[Cosmos] Preview 4

### PR DESCRIPTION
**Pull request by annatisch**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py [hunk 1]
```diff
@@ -26,19 +26,18 @@
 """
 from typing import Dict, Any, Optional
 import six
-import requests
-from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry  # pylint: disable=import-error
```
Comment (lmazuel) [inline sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py]: That's too risky on Ubuntu machines, that might blow up.
Being you depend en azure-core, that requires request 2.18 at least, you know for sure urllib3 is packaged *outside* of requests whatever platform its used.
So I would do:
```python
from urllib3.util.retry import Retry
```

File: sdk/cosmos/azure-cosmos/azure/cosmos/documents.py [hunk 2]
```diff
@@ -372,8 +372,10 @@ class ConnectionPolicy(object):  # pylint: disable=too-many-instance-attributes
     :ivar boolean UseMultipleWriteLocations:
         Flag to enable writes on any locations (regions) for geo-replicated database accounts
         in the azure Cosmos service.
-    :ivar (int or requests.packages.urllib3.util.retry) ConnectionRetryConfiguration:
-        Retry Configuration to be used for urllib3 connection retries.
+    :ivar ConnectionRetryConfiguration:
+        Retry Configuration to be used for connection retries.
+    :vartype ConnectionRetryConfiguration:
+        int or azure.cosmos.ConnectionRetryPolicy or requests.packages.urllib3.util.retry
```
Comment (lmazuel) [inline sdk/cosmos/azure-cosmos/azure/cosmos/documents.py]: int or ~azure.cosmos.ConnectionRetryPolicy or ~urllib3.util.retry.Retry

**Pull request merged**